tests/TestCase.php

```diff
@@ -3,8 +3,8 @@
 
 use ReflectionMethod;
 use Illuminate\Foundation\Application;
+use Illuminate\Database\Connectors\ConnectionFactory;
```

Comment:
To be consistent with Laravel, I keep my `use` statements in ascending order of length. This needs to be moved down a couple lines.